aspnetcore/tutorials/grpc/grpc-client.md

```diff
@@ -30,50 +30,53 @@ In this tutorial, you:
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio) to create a console app with the name *GrpcGreeterClient*.
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code) to create a console app with the name GrpcGreeterClient.
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code) to create a console app with the name *GrpcGreeterClient*.
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-on-mac-vs-full-solution) to create a console app with the name GrpcGreeterClient.
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-on-mac-vs-full-solution) to create a console app with the name *GrpcGreeterClient*.
 
 <!-- End of VS tabs -->
 
 ---
 
 ## Add required packages
 
-The following packages need to be added the gRPC client project:
+Add the following packages to the gRPC client project:
 
-* [Grpc.Core](https://www.nuget.org/packages/Grpc.Core) which contains the C# API for the C-core client.
-* [Google.Protobuf](https://www.nuget.org/packages/Google.Protobuf/) which contains protobuf message APIs for C#.
-* [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/) which contains C# tooling support for protobuf files. The tooling package isn't required at runtime, so the dependency is marked with `PrivateAssets="All"`.
+* [Grpc.Core](https://www.nuget.org/packages/Grpc.Core), which contains the C# API for the C-core client.
+* [Google.Protobuf](https://www.nuget.org/packages/Google.Protobuf/), which contains protobuf message APIs for C#.
+* [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/), which contains C# tooling support for protobuf files. The tooling package isn't required at runtime, so the dependency is marked with `PrivateAssets="All"`.
 
 Packages can be added with the following approaches:
 
 ### [Visual Studio](#tab/visual-studio)
 
-* From the **Package Manager Console** window:
-  * Go to **View** > **Other Windows** > **Package Manager Console**
-  * Navigate to the directory in which the *GrpcGreeterClient.csproj* file exists
-  * Execute the following command:
+### PMC option to install packages
+
+* From Visual Studio, select **Tools** > **NuGet Package Manager** > **Package Manager Console**
+* From the **Package Manager Console** window, navigate to the directory in which the *GrpcGreeterClient.csproj* file exists.
+* Run the following command:
 
     ```powershell
     Install-Package Grpc.Core
     ```
 
-  * Repeat for Google.Protobuf and Grpc.Tools
+* Repeat the `Install-Package` for Google.Protobuf and Grpc.Tools
+
+<!-- Tutorials shouldn't have multiple options. Select what you think is the best approach. Recommend you removed this approach. -->
```

Comment:
I copied this from https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/getting-started-with-swashbuckle.md#package-installation. But if I were to pick, we can just go with the Manage Nuget Packages option.

Reply:
Lets leave both for now and maybe have @tdykstra  update this in the future for our editorial rules..